mkdocs/docs/linux-tutorial/index.md

```diff
@@ -0,0 +1,5 @@
+# This tutorial has moved!
```

Comment:
Let's hide the nav bar on the left (and toc on the right) completely on this page:

```suggestion
---
hide:
  - navigation
  - toc
---

# This tutorial has moved!